account_fiscal_month/models/date_range_type.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class DateRangeType(models.Model):
+    _inherit = "date.range.type"
+
+    fiscal_month = fields.Boolean(string="Is fiscal month?", readonly=True)
```

Comment:
If it's readonly when is the the value assigned? Or the goal is just having the unique date.range.type created as business data as fiscal month?